doc/source/data/doc_code/transforming_datasets.py

```diff
@@ -99,7 +101,51 @@ def pandas_transform(df: pd.DataFrame) -> pd.DataFrame:
 #     'variety': 'Versicolor', 'normalized.sepal.length': 1.0}
 # -> {'sepal.width': 3.2, 'petal.length': 4.5, 'petal.width': 1.5,
 #     'variety': 'Versicolor', 'normalized.sepal.length': 0.9142857142857144}
-# __writing_native_udfs_end__
+# __writing_native_udfs_tabular_end__
+# fmt: on
+
+# fmt: off
+# __writing_native_udfs_tensor_begin__
+import ray
+import numpy as np
+
+# Load dataset.
+ds = ray.data.range_tensor(1000, shape=(2, 2))
+
+# UDF as a function on Pandas DataFrame batches.
+def tensor_transform(arr: np.ndarray) -> np.ndarray:
+    # Notice here that the ndarray is of shape
+    # (batch_size, 2, 2)
```

Comment:
Should fit a single line I think?
```suggestion
    # Notice here that the ndarray is of shape: (batch_size, 2, 2)
```